main.go

```diff
@@ -5,11 +5,25 @@ import (
 
 	"github.com/ivanterekh/go-skeleton/env"
 	"github.com/ivanterekh/go-skeleton/server"
+	"go.uber.org/zap"
 )
 
 func main() {
+	logger, err := initLogger()
```

Comment:
I don't expect any result from `init` function, I'd prefer `newLogger`, because it's actually a kind of constructor http://www.golangpatterns.info/object-oriented/constructors